docs/team-process.md

```diff
@@ -32,17 +79,39 @@ We strive to be keep our project as Agile as possible.  This means that every sp
 
 Since we strive to work in pairs and mobs, our stand ups focus more on group goals.  Every morning we meet to talk about our goals for our current sprint and also talk about our story goals for the day.  We setup **parking lot** items to give an opportunity to discuss any important topics that the entire team should be aware of.  We will often give small demos of story / bug progress to the Tax Court.
 
-### Sprint Planning 
+ - Time of Ceremony: 9:30 AM CST
+ - Frequency of Ceremony: Daily Monday through Friday
 
-TODO
+### Sprint Retrospective 
 
-### Sprint Review Prep
+The retrospective is performed at the end of the sprint and offers the team a time to reflect on what went well or what they’d like to improve. 
 
-TODO
+ - Time of Ceremony: 2:30-4:00 CST
+ - Frequency of Ceremony: Every other week Tuesday
 
-### Sprint Review
+### Sprint Planning/Refinement
 
-TODO
+We have combined our grooming and planning into one event. During this meeting we groom stories for the sprint. While a sprint is 2 weeks, we groom every week to allow for shifting PO priorities.
+
+ - Time of Ceremony: 11:30-12:00 PM CST
+ - Frequency of Ceremony: Every Tuesday 
+
+### OpEx/DevEx Backlog Review
+
+A meeting for the team to review the backlog of larger refactoring items and point for priority (importance/urgency).
+
+Refactoring backlog: https://trello.com/b/9tgrIFfA/ef-cms-opex-devex
+
+Planning poker: https://play.planningpoker.com/play/game/6sDzcKNshsnarUet2HrnTeUYR3e5tvqB 
+
+ - Time of Ceremony: 2:00-3:00 PM CST
+ - Frequency of Ceremony: Every 2 weeks on Monday 
+
+
+### Epic Refinement 
+
+ - Time of Ceremony: 2:00-3:00 PM CST
+ - Frequency of Ceremony: Every 2 weeks on Monday rint Review
```

Comment:
Isn't the frequency of this ad hoc?  For example, I'm fairly sure this has happened only once while I've been on the team.

Reply:
You're right. I changed it.